### PR TITLE
install incus on VM hosts (VMS-B32-1 only for now)

### PR DESCRIPTION
**Pull request by TimStallard**

### CODE REVIEW

File: roles/vms/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: add incus apt key
+  copy:
+    src: zabbly.asc
+    dest: /etc/apt/keyrings/zabbly.asc
+
+- name: add incus repo
+  apt_repository:
+    repo: "deb [signed-by=/etc/apt/keyrings/zabbly.asc] https://pkgs.zabbly.com/incus/lts-6.0 {{ ansible_distribution_release }} main"
```
[the remark this answers is not in the document]
Comment (TimStallard) [reply roles/vms/tasks/main.yml]: good idea, I've moved that to a variable and set 6.0-lts in role defaults for now

File: playbook.yml [hunk 2]
```diff
@@ -57,3 +57,8 @@
       tags: docker
     - role: containers_secure
       tags: containers_secure
+- name: Configure VM hosts
+  hosts: VMS-B32-1  # TODO cover all hosts once VMS-B53-1 reinstalled
+  roles:
```
[the remark this answers is not in the document]
Comment (TimStallard) [reply playbook.yml:62]: I'd really like to do this, but I think it gets a bit stuck on sown/tasks#64 as we currently hardcode those roles in many places :/

[the remark this answers is not in the document]
Comment (trickeydan) [reply playbook.yml:62]: Okay let's leave it for now and I might take a look at that other issue shortly.

[the remark this answers is not in the document]
Comment (TimStallard) [reply playbook.yml:62]: Oh! Currently we're using the `LXD Host` role for these, I misremembered and thought they were still `VMS Host` or so. Perhaps we should just add `Incus Host` then and update all the hardcoded lists.

File: roles/vms/tasks/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (trickeydan) [reply roles/vms/tasks/main.yml]: Perfect.

**Pull request merged**